src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -148,6 +148,8 @@ Also, consider doing the following:
 - Set better rate limits on the DSN keys for the projects related to the spike. 
 - If it's a specific release version that has caused the spike, add the version identifier to the project's inbound filters to avoid accepting events from that release.
 
+To review the error events dropped from Spike Protection, see your Sentry org's Settings > Subscription, Error usage section.
```

Comment:
```suggestion
To review the error events dropped because of spike protection, go to **Settings > Subscription** for your Sentry org and scroll to the "Error Usage" section of the page. 
```